inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -485,20 +458,71 @@ private void jumpToRecommendationLocationAndHighlightRecommendation(AjaxRequestT
         highlightCurrentRecommendation(aTarget);
     }
 
-    private FeatureEditor createFeatureEditor()
+    private FeatureEditor initializeFeatureEditor()
+    {
+        try {
+            createFeatureEditor();
```

Comment:
Why should createFeatureEditor switch from one document to another? It seems this method is doing too much.

Reply:
@reckart Why do you think createFeatureEditor switch from one document to another?

This method gets SourceDocument and uses SourceDocument to get AnnotationDocument and uses AnnotationDocument to get Jcas and uses Jcas to get FeatureState using the code:
```
featureState = new FeatureState(annotationFeature, (Serializable) featureSupport
            .wrapFeatureValue(annotationFeature, jCas.getCas(),
                currentRecommendation.getLabel()));
```